client/modules/User/components/Collection.jsx

```diff
@@ -181,12 +181,10 @@ class Collection extends React.Component {
   }
 
   getTitle() {
-    if (this.props.username === this.props.user.username) {
-      return this.props.t('Collection.Title');
-    }
-    return this.props.t('Collection.AnothersTitle', {
-      anotheruser: this.props.username
-    });
+    const collectionTitle = this.props.t('Collection.Title');
+    const titleSplits = collectionTitle.split('|');
+    const title = titleSplits[0].concat('| ');
+    return title;
```

Comment:
We're basically just extracting the `"p5.js Web Editor"` portion of the title, right?  We don't have any property in the translations files which is just  `"p5.js Web Editor"` but I've wondered before if we should add one.  Like `t('Common.SiteName')` or something.  FYI.

Reply:
Oh, i was thinking same. Even searched for entire i8n for it. But though i might need to discuss that before putting in.
Okay now i will implement it.